README.adoc

```diff
@@ -257,7 +260,7 @@ parts you don't use) completely before trying to install.
   * Coderay (coderay, version: 1.1.1)
   * ttfunk (ttfunk, version: 1.5.1)
   * Asciidoctor PDF (asciidoctor-pdf, version: 1.5.0)
-  * Asciidoctor Mathematical (asciidoctor-mathematical, version 0.2.2)
+  * Asciidoctor Mathematical (asciidoctor-mathematical, version 0.3.2)
```

Comment:
I tried this out myself, and installing `asciidoctor-mathematical` 0.3.2 pulled down `asciidoctor` 2.0.10.

> asciidoctor-mathematical-0.3.2 depends on asciidoctor (>= 2.0.0, ~> 2.0)

A few lines above we specify 1.5.8.  The diff of the raw HTML on the API spec from using 2.0.10 is small, suggesting that 2.0.10 isn't a problem for us, but if we go with it we should update the  `1.5.8` above.

Reply:
Thanks - I went back and checked and things appear to be working for me with `asciidoctor-mathematical` 0.2.2, which doesn't require an updated version of `asciidoctor`.  Let's go with this for now, even though `asciidoctor` 2.0.10 seems to be working OK also.

An automated travis PDF build using this config can be found [here](https://bashbaug.github.io/OpenCL-Docs/pdf/OpenCL_Env.pdf), with a properly rendered Chapter 7.